src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -49,6 +53,42 @@ public abstract class AzureServiceBusMessagePump<TMessage> : MessagePump<TMessag
         /// </summary>
         protected string Namespace { get; private set; }
 
+        /// <summary>
+        /// Gets the unique identifier for this background job to distinguish this job instance in a multi-instance deployment.
+        /// </summary>
+        public string JobId { get; }
+
+        /// <summary>
+        /// Triggered when the application host is ready to start the service.
+        /// </summary>
+        /// <param name="cancellationToken">Indicates that the start process has been aborted.</param>
+        public override async Task StartAsync(CancellationToken cancellationToken)
+        {
+            if (Settings.Options.IncludeTopicSubscription)
+            {
+                ServiceBusConnectionStringBuilder serviceBusConnectionString = await GetServiceBusConnectionStringAsync();
+
+                Logger.LogTrace("[Job: {JobId}] Creating subscription '{SubscriptionName}' on topic '{TopicPath}'...", JobId, _subscriptionName, serviceBusConnectionString.EntityPath);
```

Comment:
Do we use the `[Job: {JobId}] ` prefix for other traces as well? If not, please align this line or the others

Reply:
For all log traces, yes, but not all log messages. Like information messages doesn't have this. Do you want me to align it to every log entry that's being written in the pump?

Reply:
Let's do that, but in a follow-up PR

Reply:
Ref: https://github.com/arcus-azure/arcus.messaging/issues/58